posts/2021-11-30-RedHat-Connect-London.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "A day out in London at the RedHat Connect Summit"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/yasmin-aumeeruddy
+author_github: https://github.com/yasmin-aumeeruddy
+seo-title: A day out in London at the RedHat Connect Summit
+seo-description: 
+blog_description: 
+open-graph-image: https://openliberty.io/img/AndyMcCright-1.png
+additional_authors:
+- name: Grace Jansen
+  github: https://github.com/GraceJansen
+  image: https://avatars3.githubusercontent.com/GraceJansen
+---
+= A day out in London at the RedHat Connect Summit
+Yasmin Aumeeruddy <https://github.com/yasmin-aumeeruddy>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+link:https://twitter.com/gracejansen27[Grace Jansen] and link:https://github.com/yasmin-aumeeruddy[Yasmin Aumeeruddy] from the Open Liberty team joined our cousins at RedHat to participate in the link:https://www.redhat.com/en/summit/connect/emea/london/developers[RedHat Connect Summit in London]. This event was a single-day event, offering a variety of developer-focused sessions, many of which utilised or focused on OSS projects and initiatives. The topics of the sessions ranged from those focused on cloud-native development, to DevOps, to enterprise development, to more general language and framework sessions. As well as the conference sessions there were also lightning talks occurring in the expo area and live coding and arcade games as well! These one-day events are run in cities all over the globe. You can check out if there's an event occurring near you at their link:https://www.redhat.com/en/summit/connect#locations--dates[website].
+
+Although audiences have engaged well with virtual conferences over the past 18 months, it was great to attend an in-person event. For many, it was the first in a long time but it was a fantastic way to re-engage with the community and enable attendees to network with OSS and enterprise software developers from many different organisations.
+
+
+== Open Liberty sessions:
+Our Open Liberty advocates had two sessions accepted at this event - 'Reacting to our data driven world' and 'Replicating production in your tests with the magic of containers'. These talks were limited to 35 minutes in length but were full of great insights and demonstrations. We had a great turnout at both of these sessions, with interesting questions raised by the audience and many even staying to view our demos during their lunch break! 
```

Comment:
suggested edit:

Our Open Liberty advocates had two sessions accepted at this event - 'Reacting to our data driven world' and 'Replicating production in your tests with the magic of containers'. These talks were limited to 35 minutes in length but were full of great insights and demonstrations.
--->
Our Open Liberty advocates led two sessions at this event: _Reacting to our data driven world_ and _Replicating production in your tests with the magic of containers_. These talks were only 35 minutes long but were full of great insights and demonstrations.